doc/release-notes.md

```diff
@@ -125,12 +125,13 @@ Updated RPCs
 
 Changes to Wallet or GUI related RPCs can be found in the GUI or Wallet section below.
 
-New RPCs
---------
-
 Build System
 ------------
 
+- Release binaries are now produced using the new `guix`-based build system. Information on the details
+  is linked from the updated [/doc/release-process.md](/doc/release-process.md#build-and-attest-to-build-outputs)
+  document.
```

Comment:
For the second sentence, perhaps just: "The [/doc/release-process.md](/doc/release-process.md#build-and-attest-to-build-outputs) document has been updated accordingly."?